### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by cwehmeyer**
Here are some more corrections.

### CODE REVIEW

File: manuscript/manuscript.tex [hunk 1]
```diff
@@ -290,7 +280,17 @@ \subsection{Connecting the MSM with experimental data}
...
+
+\subsection{Advanced Methods}
+
+While the present tutorial covers Markov State Modeling 101, we encourage the user to explore other, more recent extensions of the methodology.
+Multi-ensemble Markov models (MEMMs)~\cite{dtram,tram} can be used to combine unbiased and biased simulations so as to probe kinetics of very rare events~\cite{trammbar}; MEMMs are implemented in PyEMMA.
+Recently, there have been steps towards replacing the traditional user-directed pipeline (involving featurizing, reducing dimension, discretizing, MSM estimation and coarse-graining) by a single end-to-end deep learning method such as VAMPnets~\cite{vampnet}.
+Other deep learning methods for performing the dimension reduction~\cite{tae}, finding reaction coordinates for enhanced sampling~\cite{hernandez-vde,Sultan2018-vde-enhanced-sampling,Ribeiro2018-rave}, and generative MSMs~\cite{deep-gen-msm-preprint} have been put forward and are likely to spawn an active field of research on its own right.
+Implementations of some of these methods are available or is under development in the package deeptime \url{github.com/markovmodel/deeptime}.
```
Comment (thempel) [inline manuscript/manuscript.tex]: Implementations [...] ~~is~~ under development [...]

Comment (brookehus) [reply manuscript/manuscript.tex]: I would remove "on its own right" and just end the sentence with "active field of research". if it is kept, it should be "in its own right"

File: manuscript/manuscript.tex [hunk 2]
```diff
@@ -90,9 +85,9 @@ \section{Prerequisites}
 \subsection{Background knowledge}
 \label{sec:background}
 
-For those unfamiliar with Markov state modeling, ``\emph{Markov State Models: From an Art to a Science}''~\cite{msm-brooke} provides a recent overview, while ``\emph{Markov models of molecular kinetics: Generation and validation}''~\cite{msm-jhp} describes the basic MSM theory and methodology in detail. 
+For those unfamiliar with Markov state modeling, ``\emph{Markov State Models: From an Art to a Science}''~\cite{msm-brooke} provides a recent overview, while ``\emph{Markov models of molecular kinetics: Generation and validation}''~\cite{msm-jhp} describes the basic MSM theory and methodology in detail. Two textbooks exist that focus on computational methods and applications~\cite{msm-book} and mathematical theory~\cite{schuette-sarich-book}.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "Additionally, two textbooks exist..."

File: manuscript/manuscript.tex (hunk 1 above)
Comment (brookehus) [reply manuscript/manuscript.tex]: "While the present tutorial covers Markov State Modeling 101" > "While the present tutorial is intended to cover Markov State Modeling 101"

**Pull request merged**